client/wfprev-war/src/main/angular/src/app/components/create-new-project-dialog/create-new-project-dialog.component.ts

```diff
@@ -318,4 +322,64 @@ export class CreateNewProjectDialogComponent implements OnInit {
     const validatorFn = control.validator({} as any);
     return !!validatorFn && validatorFn.hasOwnProperty(validator.name);
   }
+
+  getProjectTypeCode(value: string): string {
```

Comment:
Might be a bit cleaner to combine all these into one method, and expecting more parameters such as 
getCodeDescription(
  list: any[],
  value: string,
  matchKey: string,
  labelKey: string
){}

Not a big deal.